ArchiSteamFarm/www/pages/config.html

```diff
@@ -170,15 +170,119 @@ <h4><i class="icon fas fa-info"></i> Not implemented</h4>
                 });
             });
 
-            var objBotConfig = {}
+            function prepareBotConfigForSaving() {
+                var botName = $("#saveConfig").data("BotName");
+
+                var acceptGifts = $("#AcceptGifts").is(':checked');
```

Comment:
Hardcoding current properties of botconfig is absolutely unacceptable, this is what `/Api/Structure` and `/Api/Type`  should be used for and should be done fully dynamic.